src/test/java/com/sms/invoicify/controller/InvoiceControllerIT.java

```diff
@@ -57,7 +59,7 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
 public class InvoiceControllerIT {
 
-  private MockMvc mockMvc;
+  @Autowired private MockMvc mockMvc;
```

Comment:
Are we certain we want to Autowire this since we are now configuring mockMvc in the Before?

Reply:
Removed @autowired